_posts/blog/2019-08-29-2019-08-29-lab-meeting.md

```diff
@@ -0,0 +1,134 @@
+
+**Georgia** will be attending [eLife's Innovation Sprint](https://sprint.elifesciences.org/) next week and is wondering about tips and tricks for your first hackathon.
+Our crowd sourced advice included:
+
+* Pick a hack project you're really interested in, even if implementing it sounds challenging.
+  You'll be more motivated to learn challenging things if you're invested in the overall project.
+* Don't be afraid to say things like "I'm not confident with this" or "I don't know how to use this language".
+  Probably someone else on the team will, or no one will and you'll learn together!
+* There's always _something_ you can do that is useful to the project.
+  If you're feeling overwhelmed with the coding side, offer to write documentation instead?
+  It's often overlooked but it's the first port of call for any new user.
+* HAVE FUN! Hackathons are a great way to meet new people with similar interests to you and share knowledge.
+* Two days is a relatively short time when you factor in lunch, progress reports, "it didn't install properly", etc so always make notes for what future work others can do and why/how to do so.
+* If you have a fun idea that can't be done during the hack, set concrete actions for how it will be followed up.
+Ideas flow FAST in hackathons, but there isn't enough time to do all of them.
+* Design-hacks are totally a thing.
+  Sketch up your awesome idea (pencil and paper is fine) and think carefully about how it could work.
+  Photograph your sketches, document those ideas.
+  Maybe someone else, or future you, can implement it later.
+
+**Louise** is wondering about best ways to keep GitHub notifications organised.
+Maybe [octobox](https://octobox.io/) or [devhub](https://github.com/devhubapp/devhub)?
+
+**Ang** is interested in resouces to learn Python.
+Kirstie shared some resources from Neurohackademy:
+
+* Introduction to Python [repo](https://github.com/neurohackademy/introduction-to-python), [recording](https://neurohackademy.org/course/introduction-to-python-2/)
+* Numerical computing in python [repo](https://github.com/jbpoline/2019-numpy-tutorial), [recording](https://neurohackademy.org/course/numerical-computing-in-python/)
+* Python tips and tricks (this one is the most advanced) [repo](https://github.com/neurohackweek/python-tips-and-tricks), [recording](https://neurohackademy.org/course/python-tips-and-tricks/)
+* Visualisation with matplotlib [repo](https://github.com/KirstieJane/NH19-Visualization), [recording](https://neurohackademy.org/course/data-visualization-2/)
+
+And to finish, **Yini** started an interesting discussion about cultural differences across countries.
+Specifically, she'd been told that she hold her opinions about Chinese policies because she's been brainwashed, which she and Kirstie agreed was a very patronising and rude interpretation of her words.
```

Comment:
```suggestion
Specifically, she'd been told that she holds her opinions about Chinese policies because she's been brainwashed, which she and Kirstie agreed was a very patronising and rude interpretation of her words.
```